packages/sx05re/emulators/openbor/package.mk

```diff
@@ -42,6 +42,11 @@ makeinstall_target() {
     cp `find . -name "OpenBOR.elf" | xargs echo` $INSTALL/usr/bin/OpenBOR
     cp $PKG_DIR/scripts/*.sh $INSTALL/usr/bin
     chmod +x $INSTALL/usr/bin/*
+
     mkdir -p $INSTALL/usr/config/emuelec/configs/openbor
-	cp $PKG_DIR/config/master.cfg $INSTALL/usr/config/emuelec/configs/openbor/master.cfg
+		cp $PKG_DIR/config/master.cfg $INSTALL/usr/config/emuelec/configs/openbor/master.cfg
+		cp $PKG_DIR/config/masterff.cfg $INSTALL/usr/config/emuelec/configs/openbor/masterff.cfg
```

Comment:
Why is this change needed? did you missed the package for openborff? 

https://github.com/EmuELEC/EmuELEC/blob/dev/packages/sx05re/emulators/openbor/openborff/package.mk